Updates to trademark survey questions (#206)
rust-lang/surveys: surveys/2021-trademark-survey/questions.md

owner is endorsing or sponsoring the user. For example, when websites include
their customers’ logos on their websites, we understand that the customer’s

> logo is being used to signal that the owner of the logo approves of or endorses

Reviewer: ```suggestion
logo is being used to signal that the owner of that logo approves of or endorses
```

> the product and possibly the content of the website, and we take that

Reviewer: ```suggestion
the owner of the website, and we take that
```

> information into account when considering the product and website. A suggestion

Reviewer: ```suggestion
information into account when considering the owner of the website. A suggestion
```

Reviewer: These two sentences read odd to me and this is my attempt to maybe make it read less odd.

Author: Seems reasonable to me, in the same spirit; any of those endorsements could be possible.

>
> TODO

> ### How much of the Rust language/toolchain can someone change and still call it “Rust”?

Reviewer 2: This question somewhat implicitly assumes a fork of the existing Rust toolchain. In other words, this seems to care about what the diff is between the existing Rust language and tooling and new artifacts that come from the existing ones. I do wonder if we're missing out of subtle distinctions when someone creates an entirely new artifact and calls it Rust. I think this distinction gets at the heart of the fact that we have two groupings of things that are called "Rust": the abstract language definition which isn't really a thing but more an idea, and the actual artifacts that implement that idea (e.g., the compiler, cargo, etc).

Reviewer: I was trying to think of ways to reword this question to incorporate what I understand you to be getting at here, but it was tough given the responses we have for this question. I feel it would have to be new question of something like:

> When someone creates a new artifact for the Rust ecosystem, what is required to actually call it Rust?

The hard part here is coming up with the right choices for the survey responses.

>
> TODO

> ### Commercial businesses can use the Rust/Cargo logo on their website or social media site, in the context of an open source project that uses Rust.

Reviewer: This change definitely makes the question different than the original intent. Not choosing sides on which is better or not, but I can actually see these potentially being two separate questions.

Author: That's true. We could ask the other question ("provided they have an open source project that uses Rust"), but that seems like a much broader ask likely to get different answers. (And I was trying to avoid increasing the number of questions.)

Author: Also, for context, note that GitHub's diff here is comparing to the wrong question here. The actual change is from "Commercial businesses can use the Rust/Cargo logo on their website or social media site, provided they have an open source project that uses Rust" to "Commercial businesses can use the Rust/Cargo logo on their website or social media site, in the context of an open source project that uses Rust"

Reviewer 2: I think we're missing an "only" or "provided":

> Commercial businesses can use the Rust/Cargo logo on their website or social media site, *only* in the context of an open source project that uses Rust.

### The Rust/Cargo logo should be under a copyright license that allows modification and sharing, like a CC license:

(This does not either require or prevent the use of a trademark license with
other requirements.)

- No
- Yes, but only if it’s used for something that has nothing to do with software

> - Yes, and the license must qualify as Open Source

Reviewer 2: I'm not sure if everyone will have the same understanding of what "qualify as Open Source" means. Can we better qualify this?
